app/views/posts/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<section class="my-bio bg-secondary bg-gradient">
+    <div class="container">
+            <div class="card my-4">
+                <div class="card-header post-heading">
+                    <h4># <%=@post.Title%></h4> <h4>Comments: <%=@post.comments_count%> Likes: <%=@post.likes_count%></h4>
+                </div>
+                <div class="card-body"> <p><%=@post.text%></p></div>
+                <div class="media-info card-body">
+                    <%= render 'application/likes', post: @post, user: @user %>
+                    <ul>
+                        <li>Comments: <%=@post.comments_count%></li>
+                        <li>Likes: <%=@post.likes_count%></li>
+                    </ul>
+                </div>
+                <div class="accordion">
+                    <div class="accordion-item">
+                        <h2 class="accordion-header">
+                            <button class="accordion-button" type="button" data-bs-toggle="collapse"
+                                data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                                Comments
+                            </button>
+                        </h2>
+                        <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne">
+                            <div class="card-body">
+                                <!-- Single comment-->
+                                <%= render template: 'comments/new', post: @post, user: @user, class: "mb-3"%>
+                                <br/>
+                                <br/>
+                                <%if @post.recent_comment.count > 0%>
+                                    <% @post.recent_comment.each do |comment| %>
```

Comment:
- [ ] In the show page for a post, please render all of the comments as shown here: 
![image](https://github.com/Izebiafe/Blog/assets/91486791/d2b8b2c0-09bb-49d1-b66e-0127f95d5264)